torch/onnx/_type_utils.py

```diff
@@ -82,62 +89,136 @@ class JitScalarType(enum.IntEnum):
 
     @classmethod
     @_beartype.beartype
-    def from_name(
+    def _from_name(
```

Comment:
We documented this api in 1.13 release notes. Thoughts on adding bc_breaking?

Reply:
yes!